diff --git a/dist/Chat/ChatComposerInput.js b/dist/Chat/ChatComposerInput.js
index 5046603b4a0baf0ab3d464b9a3dedb370e2992a8..feba9688059734bce009fba0a3c76d20a139d43c 100644
--- a/dist/Chat/ChatComposerInput.js
+++ b/dist/Chat/ChatComposerInput.js
@@ -463,7 +463,7 @@ export function ChatComposerInput(props) {
       children: placeholder
     }), /*#__PURE__*/_jsx("div", {
       ref: editableRef,
-      "aria-multiline": "true",
+      "aria-multiline": triggerMenu.ariaProps.role === "textbox" ? true : undefined,
       "aria-label": label,
       contentEditable: !isDisabled,
       suppressContentEditableWarning: true,
diff --git a/src/Chat/ChatComposerInput.tsx b/src/Chat/ChatComposerInput.tsx
index 5a6a56b94ba2093c24ce0435561706c93be9dae8..7b8f097b163983c6265318808438fd157c5dc997 100644
--- a/src/Chat/ChatComposerInput.tsx
+++ b/src/Chat/ChatComposerInput.tsx
@@ -700,7 +700,7 @@ export function ChatComposerInput(props: ChatComposerInputProps) {
       )}
       <div
         ref={editableRef}
-        aria-multiline="true"
+        aria-multiline={triggerMenu.ariaProps.role === 'textbox' ? true : undefined}
         aria-label={label}
         contentEditable={!isDisabled}
         suppressContentEditableWarning
